cmd/create/cluster/cmd.go

```diff
@@ -515,6 +527,14 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	isSubnetIDsSet := cmd.Flags().Changed("subnet-ids")
+	isAvailabilityZonesSet := cmd.Flags().Changed("availability-zones")
+	// Setting subnet IDs is choosing BYOVPC implicitly,
+	// and selecting availability zones is only allowed for non-BYOVPC clusters
+	if isSubnetIDsSet && isAvailabilityZonesSet {
+		reporter.Errorf("Setting both the `subnet-ids` flag and the `availability-zones` flag is prohibited")
```

Comment:
Setting both the `subnet-ids` flag and the `availability-zones` flag is not supported